rep-2001.rst

```diff
@@ -773,6 +775,33 @@ Desktop Full
       packages: [perception, simulation, ros_ign_gazebo_demos]
 
 
+ROS Distribution Independent (September 2022 - Ongoing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These variants are not released as ROS packages within each distribution as the others are.
+Instead they are made available as independent packages outside of any ROS distribution.
+
+Build Essential
+"""""""""""""""
+
+::
+
+ - ros-build-essential:
+      packages: [cmake, git, python3-colcon-common-extensions, python3-rosdep,
+                 python3-setuptools, wget]
```

Comment:
Two comments on this variant.  First, do we want to add `build-essential` to this list?  I think we discussed this before, but I don't remember the outcome of that discussion.

Second, do we really want `python3-colcon-common-extensions` in `ros-build-essential`, or should we move it into `ros-dev-tools`?  I ask because my understanding is that we wanted `ros-build-essential` to be available for the buildfarm to depend on.  But the binary and source packaging jobs on the buildfarm do not strictly depend on colcon, so it is unnecessary dependency there.  On the other hand, PR and dev jobs probably *do* depend on colcon, so maybe we just want to take the hit on that.  Just food for thought; I don't feel strongly either way.

Reply:
```suggestion
 - ros-build-essential:
      packages: [cmake, git, python3-setuptools, wget]
```

I think these are the critical packages with the possible addition of python3 itself to support `ros_buildfarm` scripts. I'd be inclined to add `python3-ros-buildfarm` if we were using the release version but the official farm tracks HEAD.

`rosdep` and `colcon` aren't used to actually build binary packages on the build farm since by the time those builds are run on individual packages after bloom has already resolved rosdep keys to platform packages.

Reply:
```suggestion
 - ros-build-essential:
      packages: [cmake, git, python3-setuptools, wget]
```

I think these are the critical packages with the possible addition of python3 itself to support `ros_buildfarm` scripts. I'd be inclined to add `python3-ros-buildfarm` if we were using the release version but the official farm tracks HEAD.

`rosdep` and `colcon` aren't used to actually build binary packages on the build farm since by the time those builds are run on individual packages after bloom has already resolved rosdep keys to platform packages.